lib/defaults.js

```diff
@@ -8,5 +8,6 @@ module.exports = {
     'it has not had recent activity. It will be closed if no further ' +
     'activity occurs. Thank you for your contributions.',
   unmarkComment: false,
-  closeComment: false
+  closeComment: false,
+  closeStale: true // If set to false, bot will never close issues.
```

Comment:
Would it make sense to allow set `daysUntilClose` to false instead of adding a new value? So `daysUntilClose` could either be a number of days, or false to disable. 

I don't feel strongly either way, but I'm suggesting it since it is mutually exclusive with this value, so you'd likely want to delete or comment out that value if you set this one to false.

If you think it makes sense to add this new value, it should also be documented in the `README`.

Reply:
I'm +1 for `daysUntilClose` to false. That mirrors what's in #79 to disable locking, and doesn't add more unused configs.